### PR TITLE
HTBHF-503 Created a privacy notice page

### PR DESCRIPTION
**Pull request by YetAnotherMatt**

### CODE REVIEW

File: src/test/acceptance/features/privacy-notice.feature [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Feature: Privacy notice page
```
Comment (gethyn1) [inline src/test/acceptance/features/privacy-notice.feature]: Should this be *privacy policy*? Or is that the agreed language? *privacy notice* sounds like a popup or banner (subjective I guess).

File: src/test/acceptance/features/privacy-notice.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  In order to facilitate the beta HTBHF programme
+  As a potential claimant
+  I want to see the details of how my details are used and retained
+
+  Scenario: The Privacy notice page is accessible
+    Given I navigate to the HTBHF overview page
```
Comment (mikeSmith08) [inline src/test/acceptance/features/privacy-notice.feature]: Might it be prudent for this to not be the Overview page and rather the enter name page as the Overview page will disappear at some point?

File: src/test/acceptance/features/privacy-notice.feature (hunk 1 above)
Comment (YetAnotherMatt) [reply src/test/acceptance/features/privacy-notice.feature]: That's what it is called in the prototype - i.e. in the content from Kate

File: src/test/acceptance/features/privacy-notice.feature (hunk 2 above)
Comment (YetAnotherMatt) [reply src/test/acceptance/features/privacy-notice.feature]: Fair point. There is then no difference between this and the last scenario - shall I remove this one? The same would apply for the cookies feature.

Comment (mikeSmith08) [reply src/test/acceptance/features/privacy-notice.feature]: Yes, I believe so.

File: src/test/common/steps/privacy-notice-steps.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  await checkPrivacyNoticePageContentIsPresentAndCorrect()
+})
+
+Then(/^the back link on the privacy-notice page links to the enter name page$/, async function () {
+  const backLink = await pages.privacyNotice.getBackLink()
+  const href = await backLink.getAttribute('href')
+
+  // using startsWith() instead of equals() so query parameters do not fail the test
+  const enterNameUrl = `${pages.url}${pages.enterName.getPath()}`
+  expect(href.startsWith(enterNameUrl)).to.equal(true)
```
Comment (mikeSmith08) [inline src/test/common/steps/privacy-notice-steps.js]: You can use Chai's `startWith`, so something like: `expect('abcdef').to.startsWith('abc')` or `'abcdef'.should.startWith('abc')` if the grammar of the first doesn't seem quite right.

Comment (YetAnotherMatt) [reply src/test/common/steps/privacy-notice-steps.js]: Gethyn thinks it requires bringing in a new package - not worthwhile for this one assertion

Comment (mikeSmith08) [reply src/test/common/steps/privacy-notice-steps.js]: That's fine.

**Pull request merged**